### PR TITLE
add sci-m docs

### PR DESCRIPTION
**Pull request by jmcook1186**
### Types of changes

- [x] Enhancement (project structure, spelling, grammar, formatting)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### A description of the changes proposed in the Pull Request

Updates the docs for sci-m and adds an example `impl` file that runs `sci-m` in isolation.

### CODE REVIEW

File: examples/impls/sci-m.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      config:
+        sci-m:
+          te: 1533.120 # gCO2eq
+          tir: 1 # s per hour
+          el: 3 # 3 years in seconds        
+          rr: 1
+          tor: 8
+      observations: 
+        - timestamp: 2023-07-06T00:00
+          duration: 3600
```
Comment (narekhovhannisyan) [inline examples/impls/sci-m.yml:24]: newline at the end of file

**Comment by narekhovhannisyan** [comment]
@jmcook1186 for commit messages to be according to our contribution document, you can use following pattern

```
scope: change which is done.

// f. ex. for current changes
docs: add `sci-m` documentation.
```

File: examples/impls/sci-m.yml (hunk 1 above)
Comment (jmcook1186) [reply examples/impls/sci-m.yml:24]:
```suggestion
          duration: 3600

```

File: examples/impls/sci-m.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+graph:
+  children:
+    child:
+      pipeline: 
+        - sci-m # duration & config -> embodied
+      config:
+        sci-m:
+          te: 1533.120 # gCO2eq
+          tir: 1 # s per hour
+          el: 3 # 3 years in seconds
```
Comment (gnanakeethan) [inline examples/impls/sci-m.yml:19]:
```suggestion
          el: 94608000 # 3 years in seconds        
```

File: examples/impls/sci-m.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      path: ''
+graph:
+  children:
+    child:
+      pipeline: 
+        - sci-m # duration & config -> embodied
+      config:
+        sci-m:
+          te: 1533.120 # gCO2eq
+          tir: 1 # s per hour
```
Comment (gnanakeethan) [inline examples/impls/sci-m.yml:18]:
```suggestion
          tir: 3600 # 1hr in seconds
```

**Review by gnanakeethan — CHANGES_REQUESTED** [review]

**Review by gnanakeethan — APPROVED** [review]

**Pull request merged**